schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/update-fixed-info.xsl

```diff
@@ -318,13 +318,6 @@
     </xsl:copy>
   </xsl:template>
 
-
-  <!-- The topic category directive may create empty topic. -->
```

Comment:
Why is this template removed?

Reply:
Because it was added as a workaround to the fact that "Inserting a snippet in an element which does not exist could create an additional self closing tag". The directive for topics category is using the XPath insert API and is this case.